### PR TITLE
[1.16] Setting Kafka Webhook HPA to minReplica of two

### PR DESCRIPTION
**Pull request by matzew**
As per title, done for SRVKE-884

### CODE REVIEW

File: knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml [hunk 1]
```diff
@@ -957,7 +957,7 @@ spec:
     apiVersion: apps/v1
     kind: Deployment
     name: kafka-webhook
-  minReplicas: 1
+  minReplicas: 2
```
Comment (markusthoemmes) [inline knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml:960]: Do we really want this? It'll mean users won't be able to tune this down to 1 :thinking:.

Comment (matzew) [reply knative-operator/deploy/resources/knativekafka/1-channel-consolidated.yaml:960]: I think I will anyways tackle this differently

**Pull request closed without merging**